plugins/registry.js

```diff
@@ -904,4 +904,11 @@ export default class PluginRegistry {
             },
         });
     }
+
+    forceTeamSidebarToBeVisible(showSidebar) {
+        store.dispatch({
+            type: ActionTypes.FORCE_TEAM_SIDEBAR_TO_BE_VISIBLE,
+            showSidebar,
+        });
+    }
```

Comment:
This is interesting because the functions on the registry are generally functions that 
"register" something, rather than just firing off an action like this. I suppose it seems fine to me. @hmhealey What are your thoughts on this?